airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -361,14 +367,32 @@ public WebBackendConnectionRead webBackendUpdateConnection(final WebBackendConne
     connectionRead = connectionsHandler.updateConnection(connectionUpdate);
```

Comment:
I believe that I found a bug here while running the acceptance test. Here we are saving the connectionUpdate. This is will also update the catalog since it is part of the update object. So when we fetch it on line 382, the existed one wil thus have no diff with the updated one because the update have already happen. https://github.com/airbytehq/airbyte/pull/14406/commits/58319dd42c113837e9abfadda2e7ef66c422ea46 fixed my acceptance test. cc @lmossman for your own acceptance test